flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+          x-field-uid: 4
+    Flow.RSVPRecordRouteIPv4.Flag:
+      type: object
+      properties:
+        choice:
+            type: string
+            default: local_protection_available
+            x-enum:
+              local_protection_available:
+                x-field-uid: 1
+              local_protection_in_use:
+                x-field-uid: 2
+            x-field-uid: 1   
+    Flow.RSVP.PathRecordRouteType1Label:
+      description: >-
+        Class = RECORD_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: Label, C-Type: 3
+      type: object
+      properties:
+        length:
+          description: >-
+              The Length contains the total length of the subobject in bytes, including the Type and Length fields.  
+              The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPRouteRecord.Length'
+          x-field-uid: 1
+        flags:
+          x-field-pattern:
+            description: >-
+              0x01 = Global label. This flag indicates that the label will be understood if received on any interface.
+            format: integer
+            length: 8
+            default: 1
+          x-field-uid: 2 
+        c_type:
+          x-field-pattern:
+            description: >-
+              The C-Type of the included Label Object. Copied from the Label object.
+            format: integer
+            length: 8
+            default: 0
+          x-field-uid: 3
+        contents_of_label_obejct:
+          description: >-
+            The contents of the Label Object. Copied from the Label Object.
+          type: string
+          format: hex
+          default: "00"
+          x-field-uid: 4
+    Flow.RSVPRouteRecord.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
+            If the OTG is unable to generate a value the default value must be used.
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 2
+        value:
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 3 
+    Flow.RSVP.PathObjectsCustom:
+      type: object
+      description: Custom packet header
+      required: [bytes]
+      properties:
+        type:
+          x-field-pattern:
+            description: >-
+              User defined object type.
+            format: integer
+            length: 8
+            default: 0
+            features: [count]
+          x-field-uid: 1
+        length:
```

Comment:
Refer to earlier comment on inner vs outer length

Reply:
Moved length to individual object level so that each object including "custom" uses the inner length.